bindings/apns/authorization_builder.go

```diff
@@ -17,7 +17,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/golang-jwt/jwt"
+	jwt "github.com/golang-jwt/jwt/v4"
```

Comment:
The package at /jwt has been deprecated and now they use versioning. Despite the version increase, it's fully backwards-compatible. Other parts of dapr already use /jwt/v4